### ISSUE TITLE
Error: [ngTransclude:orphan] in versions > 1.0.5

### ISSUE THREAD

**Issue by f1tech** [angular/material#8340]
If I use any version of AM > 1.0.5 I receive an Error: [ngTransclude:orphan] error for any <md-button> included in any state after my initial state is loaded and I also receive this error on buttons that remain on the page that did not previously throw it. 

A sample complete error message and call stack looks like this:

angular.js:13550 Error: [ngTransclude:orphan] http://errors.angularjs.org/1.5.5/ngTransclude/orphan?p0=%3Cbuttonlass%3D%22md-icon-button%20md-button%20md-ink-ripple%22%20type%3D%22button%22%20ng-transclude%3D%22%22%20ng-click%3D%22queue.openOrder()%22%20aria-label%3D%22mode_edit%22%3E
    at Error (native)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:6:412
    at La.link (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:303:226)
    at ja (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:80:350)
    at n (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:66:28)
    at g (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:58:305)
    at n (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:65:473)
    at g (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:58:305)
    at n (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:65:473)
    at g (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js:58:305)

This is the syntax I am using for my buttons:

```
    <md-button class="md-icon-button" ng-click="queue.openOrder()">
        <md-tooltip md-direction="right">
            Open Order
        </md-tooltip>
        <i class="material-icons md-dark">mode_edit</i>
    </md-button>

```
I'm using Angular 1.5.5 on the latest version of Chrome for Windows.

**Comment by jayeshanandani** [comment]
Can you try this with 1.1.0-rc-4 AM?

**Comment by f1tech** [comment]
I'm not sure if I have the same build you reference above, but using: https://rawgit.com/angular/bower-material/master/angular-material.js (v1.1.0-rc4-master-73741db) I am seeing the same behavior.

**Comment by crisbeto** [comment]
I don't get that button running the master branch. Are you sure there's not something wrong with your template? Here's what Angular error says:

```
Illegal use of ngTransclude directive in the template! No parent directive that requires a transclusion found. Element: <buttonlass="md-icon-button md-button md-ink-ripple" type="button" ng-transclude="" ng-click="queue.openOrder()" aria-label="mode_edit">
```

Note the node name.

**Comment by ThomasBurleson** [comment]
Invalid issue without CodePen. Closing as not reproducible.

**Comment by f1tech** [comment]
I have a pretty complicated scenario. I doubt I could reproduce it with a codepen, but it still occurs. It's only occurring where I use the components introduced in Angular 1.5.

**Comment by fxding** [comment]
I think this problem is because use `md-button` in `ng-repeat`. I have the same problem, and I use button to replace it.

**Comment by MarcBernat** [comment]
Solution: [Link](http://stackoverflow.com/questions/28725217/md-button-inside-ui-gmap-window-angular-google-maps-error)
